proto/osmosis/incentives/genesis.proto

```diff
@@ -3,15 +3,13 @@ package osmosis.incentives;
 
 import "gogoproto/gogo.proto";
 import "osmosis/incentives/params.proto";
+import "osmosis/incentives/pot.proto";
 
 option go_package = "github.com/c-osmosis/osmosis/x/incentives/types";
 
 // GenesisState defines the incentives module's genesis state.
 message GenesisState {
     // params defines all the parameters of the module
     Params params = 1 [(gogoproto.nullable) = false];
-    // current epoch
-    int64 current_epoch = 2 [(gogoproto.moretags) = "yaml:\"current_epoch\""];
-    // current epoch begin block
-    int64 epoch_begin_block = 3 [(gogoproto.moretags) = "yaml:\"epoch_begin_block\""];
+    repeated Pot pots = 2 [(gogoproto.nullable) = false];
```

Comment:
This fixes a bug with pots not getting put into genesis?

Reply:
yes